plugins/tiddlywiki/highlight-legacy/readme.tid

```diff
@@ -0,0 +1,81 @@
+title: $:/plugins/tiddlywiki/highlight/readme
+
+This plugin provides syntax highlighting of code blocks using v9.18.1 of [[highlight.js|https://github.com/isagalaev/highlight.js]] from Ivan Sagalaev.
```

Comment:
The readme is what users will see in the plugin library listing, so it's important that the very first paragraph explains that this is the legacy version, and clearly explains the circumstances under which it should be used.

Reply:
done